sycl/CMakeLists.txt

```diff
@@ -190,17 +190,11 @@ add_custom_target( sycl-toolchain
           llvm-objcopy
           sycl-post-link
           sycl-ls
+          opencl-aot
```

Comment:
It's quite philosophical question... currently this tool is essential part of SYCL AOT mode so I was thinking it might be ok to just add it to the list here that's forming a full package. Same question can be asked actually about other components listed here. Let's see if there are strong objections from others or some other rationale why it should not be here.

Reply:
opencl-aot is an optional component while [most of] others are required to build any SYCL application.

Reply:
I've looked closer. sycl-toolchain target defined here doesn't really have "sycl core" meaning, there are many dependencies added later in the CMakeLists.txt code into sycl-toolchain, making it basically a topmost target for the whole package.
If we want to restructure this better I can do this as a separate patch outlining clearly core component vs. optional ones.